src/Events/ContentCreating.php

```diff
@@ -0,0 +1,23 @@
+<?php
+ 
+namespace Portable\FilaCms\Events;
+ 
+use Portable\FilaCms\Models\Page;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+use Str;
+
+class ContentCreating
+{
+    use Dispatchable, InteractsWithSockets, SerializesModels;
+ 
+    /**
+     * Create a new event instance.
+     */
+    public function __construct( public Page $page ) {
+        $page->created_user_id = auth()->user()->id;
+        $page->updated_user_id = auth()->user()->id;
```

Comment:
Can we make sure we're only doing this if the slug is blank?  The user should potentially be able to override the slug.